source/ksplocations_example.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <PropertyGroup>
+    <!-- Edit these to point to the files as you find them on your machine -->
+    <TextTransformExe>C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\texttransform.exe</TextTransformExe>
+    <Pdb2MdbExe>c:\pdb2mdb\pdb2mdb.exe</Pdb2MdbExe>
```

Comment:
This is out of date.  Should be:

`C:\Program Files\Unity\Editor\Data\MonoBleedingEdge\lib\mono\4.5\pdb2mdb.exe`